.github/release-drafter.yml

```diff
@@ -0,0 +1,4 @@
+template: |
+  # Changelog
+
+  $CHANGES
```

Comment:
Do we want to specify any specific format or a labelling technique based format , so that major, minor, features are listed fine?